pkg/packagekit/updates.css

```diff
@@ -66,6 +66,34 @@ tr.security {
     border: none;
 }
 
+.flow-list-blank-slate {
+    padding-top: 8em;
+    margin: 0 auto;
+    max-width: 68rem;
+    text-align: center;
+}
+
+.flow-list {
+    margin: 1em 1em 3em;
+    padding: 0;
+    width: 100%;
+    max-width: 68rem;
+    text-align: left;
+    box-sizing: border-box;
+}
+
+.flow-list li {
+    text-align: left;
+    box-sizing: border-box;
+    width: 30ex;
```

Comment:
I suggest changing this to `22rem` and changing `max-width` of `68rem` above to `66rem`.

Or go for `23rem` here and `69rem` above.